### ISSUE TITLE
Uppercase keyspace causes Kong migrations to fail

### ISSUE THREAD

**Issue by montanaflynn** [Mashape/kong#249]
```shell
$ kong start
[INFO] Using configuration: /etc/kong/kong.yml
[INFO] Proxy port.........8000
       Admin API port.....8001
       Database...........cassandra keepalive=60000 port=9042 timeout=1000 hosts=kongdb.org keyspace=Yoda
[INFO] Database not initialized. Running migrations...
[ERR] Cassandra error: Cassandra returned error (Invalid): "Keyspace 'yoda' does not exist"
```
The keyspace is lowercase in the error so I changed it to lowercase in `kong.yml` and it then worked.

**Comment by thibaultCha** [comment]
Actually migrations are successful but a keyspace will always be lower case, so a fix would be to force it to be lower case in the configuration loading I guess

**Comment by thibaultCha** [comment]
Sorry, I meant the keyspace creation is successful*

**Comment by thefosk** [comment]
@thibaultCha is this closed?

**Comment by thibaultCha** [comment]
No

**Comment by thibaultCha** [comment]
Ok fixed by #271!
